pkg/volumes/do/volumes.go

```diff
@@ -136,54 +155,136 @@ func (a *DOVolumes) findVolumes(filterByRegion bool) ([]*volumes.Volume, error)
 	var myvolumes []*volumes.Volume
 	for _, doVolume := range doVolumes {
 
-		glog.V(2).Infof("Iterating DO Volume with name=%s and ID=%s", doVolume.Name, doVolume.ID)
+		glog.V(2).Infof("Iterating DO Volume with name=%s ID=%s, nametag=%s", doVolume.Name, doVolume.ID, a.nameTag)
 
-		// determine if this volume belongs to this cluster
-		// check for string a.ClusterName but with strings "." replaced with "-"
-		// example volume name will be like kops-1-etcd-events-dev5-techthreads-co-in
-		var k8sclusterName = a.matchTags["kubernetes.io/cluster"]
-
-		if !strings.Contains(doVolume.Name, strings.Replace(k8sclusterName, ".", "-", -1)) {
-			glog.V(2).Infof("DO Volume with name=%s does not match our cluster name=%s", doVolume.Name, k8sclusterName)
-			continue
+		for _, volumeTag := range doVolume.Tags {
+			glog.V(2).Infof("Iterating all Volume tags for volume name=%s volumetag=%s", doVolume.Name, volumeTag)
```

Comment:
Nit: you can probably just print the volumeTag slice, it is printed reasonable:

`Infof("iterating all volume tags for volume %q, tags %v", doVolume.Name, doVolume.Tags)`

Reply:
Sure, done.